apl-db-updater/src/main/resources/db/migration/h2/apl/V1_0__Initialize_tables.sql

```diff
@@ -729,7 +695,7 @@ CREATE TABLE IF NOT EXISTS `order_scan`
 
 CREATE TABLE IF NOT EXISTS `peer`
 (
-    `address`      varchar(191) COLLATE utf8_bin NOT NULL,
+    `address`      varchar(191)  NOT NULL,
```

Comment:
There is a difference between collations
https://www.programmersought.com/article/6240397587/